rtv/packages/praw/__init__.py

```diff
@@ -841,7 +842,7 @@ def get_domain_listing(self, domain, sort='hot', period=None, *args,
 
         :param domain: The domain to generate a submission listing for.
         :param sort: When provided must be one of 'hot', 'new', 'rising',
-            'controversial, or 'top'. Defaults to 'hot'.
```

Comment:
The changes to praw will be need to be made in my fork here https://github.com/michael-lazar/praw3 and then pulled in using the ``scripts/update_packages.py`` script. I can take care of that though.